validphys2/src/validphys/config.py

```diff
@@ -1505,6 +1505,19 @@ def produce_group_dataset_inputs_by_metadata(
             for name, group in res.items()
         ]
 
+    def produce_fivetheories(self,point_prescription):
+        if point_prescription == "5bar point":
+            return 'bar'
+        elif point_prescription == "5 point":
+            return 'nobar'
+        return None 
+    
+    def produce_seventheories(self,point_prescription):
+        if point_prescription == "7 point":
+            return None
+        elif point_prescription == "7original point":
+            return 'original'
+        return None
```

Comment:
```suggestion
    def produce_fivetheories(self, point_prescription):
        if point_prescription == "5bar point":
            return "bar"
        elif point_prescription == "5 point":
            return "nobar"
        return None 
    
    def produce_seventheories(self, point_prescription):
        if point_prescription == "7 point":
            return None
        elif point_prescription == "7original point":
            return "original"
        return None
```

Why is the first `None` in `7 point` correct?

Reply:
because here 
https://github.com/NNPDF/nnpdf/blob/16e2ac648853ddadf68feb23d288f243c65d5126/validphys2/src/validphys/theorycovariance/construction.py#L395
the Gavin version of the 7pt prescription is the default one. So it is executed when seventheories is None

Reply:
Ah, right. I remember now. Could you add a comment? (so that when I get confused again after two weeks I know the answer)

Reply:
yes, of course